operator/tests/central/basic/100-assert.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
```

Comment:
So, the number is equivalent to the applied update version on the CR.
Since we create a new version here, I chose to bump the number, and keep the deletion step as the last step applied, hence bumping this one to `100-...`.